Makefile

```diff
@@ -1,6 +1,9 @@
 BUILD_TOOL = xcodebuild
-BUILD_SCHEME = SQLite Mac
```

Comment:
I think it would need to be wrapped in an `if [ -z "$BUILD_SCHEME" ]` to make it act like a default instead of an override.